package.json

```diff
@@ -29,7 +30,7 @@
   "devDependencies": {
     "@babel/core": "^7.13.10",
     "@mozilla-protocol/core": "^12.1.0",
-    "@mozilla/glean": "^0.7.0",
+    "@mozilla/glean": "^0.8.0",
```

Comment:
0.8.1 is the newest version, important to update because it contains a critical bugfix.

Reply:
Good catch.